core_lib/Cargo.toml

```diff
@@ -45,7 +45,7 @@ prost-build = "0.12"
 
 [features]
 default = ["experimental"]
-experimental = ["bluer"]
```

Comment:
Leaving it in completely breaks cargo build

> don't remove it from the experimental, this would break Linux build I guess

If I leave it in it completely breaks cargo build

Reply:
would this be okay?

```rust
[target.'cfg(target_os = "linux")'.dependencies]
bluer = { version = "0.17.0", features = ["full"], optional = true }
```

Reply:
Yes that's what I was going to recommend you :)